From f4f0b39a45ba9852bea920f82d5b5876fb6451ea Mon Sep 17 00:00:00 2001
From: Scott Weaver <scweaver@redhat.com>
Date: Wed, 13 Mar 2024 13:19:02 -0400
Subject: [PATCH] lsm: update security_lock_kernel_down

Update to RHEL-only security_lock_kernel_down() due to 260017f3 ('lsm:
use default hook return value in call_int_hook()').

Signed-off-by: Scott Weaver <scweaver@redhat.com>
---
 security/security.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/security.c b/security/security.c
index 57798d994a01..e46afc5ca171 100644
--- a/security/security.c
+++ b/security/security.c
@@ -5812,7 +5812,7 @@ EXPORT_SYMBOL(security_locked_down);
  */
 int security_lock_kernel_down(const char *where, enum lockdown_reason level)
 {
-	return call_int_hook(lock_kernel_down, 0, where, level);
+	return call_int_hook(lock_kernel_down, where, level);
 }
 EXPORT_SYMBOL(security_lock_kernel_down);
 
-- 
2.47.1

